### ISSUE TITLE
the accuracy decrease appearently after converting caffe to keras.

### ISSUE THREAD

**Issue by PengBY91** [MarcBS/keras#52]
I have converted a model in caffe to Keras model using your projection. The caffe model on LFW has a accuracy of 98.2%, but after converting to keras, the accuracy on LFW is only 80%. The pre-processing methods for both model are the same. I have also tried other converting code to convert caffe model to keras, but all the results are similar, the accuracy decreases to 77%~ 80%.
I don't know why this would happen. Is there any caution I should care? Or some others I have missed?
Help for your response, thanks!

**Comment by PengBY91** [comment]
I made a mistake about the difference between the input shape of caffe and keras. Now this problem is solved.
